sql/expression/arithmetic.go

```diff
@@ -0,0 +1,297 @@
+package expression
+
+import (
+	"fmt"
+
+	"github.com/spf13/cast"
+	"gopkg.in/src-d/go-vitess.v0/vt/sqlparser"
+
+	"gopkg.in/src-d/go-mysql-server.v0/sql"
+)
+
+// Arithmetic expressions (+, -, *, /, ...)
+type Arithmetic struct {
+	BinaryExpression
+	op string
+}
+
+// NewArithmetic creates a new Arithmetic sql.Expression.
+func NewArithmetic(left, right sql.Expression, op string) *Arithmetic {
+	return &Arithmetic{BinaryExpression{Left: left, Right: right}, op}
+}
+
+// NewPlus creates a new Arithmetic + sql.Expression.
+func NewPlus(left, right sql.Expression) *Arithmetic {
+	return NewArithmetic(left, right, sqlparser.PlusStr)
```

Comment:
Should we depend on go-vitess for the arithmetics strings or should we have our own constants for this?

Reply:
@mcarmonaa - I had the same thought, but we already rely on `sqlparser` consts (`ast.go`). in `parse.go`, so if we introduce another one we'll have to think about sync it up in case of any potential future changes. So I decided to use the predefined consts. everywhere for consistency.